.gitignore

```diff
@@ -47,3 +47,6 @@ yarn.lock
 package-lock.json
 
 css/styles.css
+
+# Cache
+.cache/
```

Comment:
what is this a cache of? what produces it?

Reply:
I'm not sure what produces it, it just always appears in my staged files whenever I work in react dates. Currently the folder contains one file whose contents are:
```{"value":{"success":true,"data":{"next":{"version":"5.0.0-rc.6","info":{"plain":"### Bug Fixes\n\n* Addon-actions: FIX performance by upgrading to telejson 2.1 ([#5751](https://github.com/storybooks/storybook/pull/5751))\n* UI: FIX bad treeview mockdata ([#5741](https://github.com/storybooks/storybook/pull/5741))\n* UI: About page styling fixes ([#5732](https://github.com/storybooks/storybook/pull/5732))\n* UI: Restore the toolbar eject button ([#5737](https://github.com/storybooks/storybook/pull/5737))"}},"latest":{"version":"4.1.13","info":{"plain":"### Documentation\n\n* Fix unkind addon-ondevice-knobs readme. ([#5629](https://github.com/storybooks/storybook/pull/5629))\n\n### Maintenance\n\n* Cleanup CLI version notice ([#5699](https://github.com/storybooks/storybook/pull/5699))\n* Use static versions.json file instead of hacking one in ([#5675](https://github.com/storybooks/storybook/pull/5675))"}}},"time":1551129533811},"type":"Object"}```

Reply:
hm, maybe something in storybook. cool to add then!